contracts/Token.sol

```diff
@@ -1,39 +0,0 @@
-// SPDX-License-Identifier: MIT
-pragma solidity >=0.6.0 <0.8.0;
```

Comment:
Please, revert this SC back. This is Cere ERC20 SC. We can not remove/update it.

Reply:
okay.